flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -351,8 +351,20 @@ if [ -z "$HADOOP_CONF_DIR" ]; then
     fi
 fi
 
+# try and set HADOOP_CONF_DIR to some common default if it's not set
+if [ -z "$HADOOP_CONF_DIR" ]; then
+    if [ -d "/etc/hadoop/conf" ]; then
+        HADOOP_CONF_DIR="/etc/hadoop/conf"
```

Comment:
I would suggest to print a message to the user that we are using this HADOOP_CONF_DIR